meta-mender-demo/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -20,4 +20,7 @@ do_install_append() {
         install -m 0755 ${WORKDIR}/eth.network ${D}${sysconfdir}/systemd/network
         install -m 0755 ${WORKDIR}/enp.network ${D}${sysconfdir}/systemd/network
   fi
+
+  # Remove getty@tty1.service
+  rm ${D}${sysconfdir}/systemd/system/getty.target.wants/getty@tty1.service
```

Comment:
This does not seem to work, at least not on RPi4. I do get the console on the HDMI output and I have to manually start psplash for it to show.

Reply:
It seems we have to set `SPLASH = "psplash"` ? (https://github.com/agherzan/meta-raspberrypi/blob/warrior/recipes-core/images/rpi-basic-image.bb#L9)

Reply:
Yes, and is it necessary? I thought the idea was to run psplash until the login prompt is ready and then switch to the prompt? Certainly I don't think we want to leave the login disabled, since the user may prefer to use the HDMI output to log in.

Reply:
> I don't think we want to leave the login disabled, since the user may prefer to use the HDMI output to log in.

You're indeed right, I will update accordingly.